database/sql/src/main/scala/cromwell/database/slick/MetadataSlickDatabase.scala

```diff
@@ -43,6 +43,11 @@ class MetadataSlickDatabase(originalDatabaseConfig: Config)
     runTransaction(action)
   }
 
+  override def metadataSummaryEntryExists(workflowExecutionUuid: String)(implicit ec: ExecutionContext): Future[Boolean] = {
+    val action = dataAccess.workflowMetadataSummaryEntryExistsForWorkflowExecutionUuid(workflowExecutionUuid).result
```

Comment:
I think you could drop the `ForWorkflowExecutionUuid` since that’s already the name of the parameter